app/uk/gov/hmrc/voapropertylinking/controllers/PropertyRepresentationController.scala

```diff
@@ -125,4 +125,14 @@ class PropertyRepresentationController @Inject()(
         }
     }
   }
+
+  def unassignAgent(): Action[JsValue] = authenticated.async(parse.json) { implicit request =>
+    withJsonBody[UnassignAgent] { appointAgent =>
+      organisationManagementApi
+        .agentAppointmentChanges(AppointmentChangesRequest(appointAgent))
+        .map { response =>
+          Accepted(Json.toJson(response))
```

Comment:
Looks like should be Ok(...) not Accepted. Is the action not finished by the time this endpoint replies? Are they still processing some queues asynchronously?

Reply:
scratch that, Dan confirmed this is an async endpoint. Cool